server/tools/peertube-import-videos.ts

```diff
@@ -156,7 +156,7 @@ function processVideo (info: any, languageCode: string, cwd: string, url: string
       return res()
     }
 
-    const path = join(cwd, createHash('sha256').update(program['targetUrl']).digest('hex') + '.mp4')
+    const path = join(cwd, sha256(program['targetUrl']) + '.mp4')
```

Comment:
You could use the `url` parameter instead (in case we import a channel or a playlist)